config/peerpods/podvm/aws-podvm-image-handler.sh

```
@@ -200,13 +206,38 @@ function bootc_to_ami() {
+    else
+        error_exit "bootc_to_ami failed: no valid credentials are properly configured"
+    fi
+
     bib_args="--type ami --aws-ami-name ${aws_ami_name} --aws-bucket ${aws_bucket} --aws-region ${AWS_REGION}"
     bootc_image_builder_conversion "${container_image_repo_url}" "${image_tag}" "${auth_json_file}" "${run_args}" "${bib_args}"
 }
 
+# Check if peer-pods-secret is based on CCO-created secret (needs permissions extension) or otherwise (should have full permissions).
+# Returns 0 if CCO flow, 1 otherwise (should have full permissions).
+function is_cco_flow_secret() {
+    local cco_label=$(kubectl get secret peer-pods-secret \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)